docker-compose.yaml

```diff
@@ -0,0 +1,53 @@
+version: '3.1'
+
+services:
+
+    mysql:
+        container_name: foundry_mysql
+        image: mysql
+        command: --default-authentication-plugin=mysql_native_password
+        environment:
+            MYSQL_ROOT_PASSWORD: root
+            MYSQL_DATABASE: zenstruck_foundry
+        ports:
+            - "3306:3306"
+        healthcheck:
+            test: 'mysql -u root -proot -D zenstruck_foundry -e "SELECT 1"'
+            timeout: 20s
+            retries: 10
+            interval: 2s
+
+    postgres:
+        container_name: foundry_postgres
+        image: postgres:13
+        environment:
+            POSTGRES_DB: zenstruck_foundry
+            POSTGRES_PASSWORD: root
+            POSTGRES_USER: root
+        ports:
+            - "5432:5432"
+        volumes:
+            - db-data:/var/lib/postgresql/data:rw
+        healthcheck:
+            test: /usr/bin/pg_isready
+            timeout: 10s
+            retries: 10
+
+    mongo:
+        container_name: foundry_mongo
+        image: mongo
+        tmpfs:
+            - /data
+        environment:
+            - MONGO_INITDB_ROOT_USERNAME=admin
+            - MONGO_INITDB_ROOT_PASSWORD=admin
+            - MONGO_INITDB_DATABASE=mongo
+            - MONGO_NON_ROOT_USERNAME=mongo
+            - MONGO_NON_ROOT_PASSWORD=mongo
```

Comment:
mongo is a PITA to config, I did not manage to have the same credentials than for sql containers